api/src/main/java/org/openmrs/module/htmltojson/htmltojson/HtmlObsTagExtractor.java

```diff
@@ -20,26 +21,32 @@ public static HtmlFormDataPoint extractObsTag(Element obsTag) {
 			return null;
 		}
 		
+		String conceptName = concept.getName().getName();
+		
 		HtmlFormDataPoint dataPoint = new HtmlFormDataPoint();
 		String requiredField = obsTag.attr("required");
 		String idAttr = obsTag.attr("id");
 		String obsLabel = HtmlFormUtil.extractQuestionLabel(obsTag);
 		
-		dataPoint.setQuestionLabel(obsLabel);
+		String questionLabel = obsLabel.isEmpty() || obsLabel.equals("\\") ? conceptName : obsLabel;
+		
+		dataPoint.setQuestionLabel(WordUtils.capitalizeFully(questionLabel));
```

Comment:
Do we want to capitalize the labels?

Reply:
@ojwanganto yes, some labels are not capitalised (the first letter of each word in the sentence), I seen users doing these updates manually. But if you want I can put this behind configuration

Reply:
I have no objection. Makes sense